src/test/java/org/prebid/server/bidder/bizzclick/BizzclickBidderTest.java

```diff
@@ -0,0 +1,401 @@
+    }
+
+    @Test
+    public void makeHttpRequestsShouldCreateSingleRequestWithExpectedUri() {
+        // given
+        final BidRequest bidRequest = givenBidRequest(givenImp("account id", "placement id"));
+
+        // when
+        final Result<List<HttpRequest<BidRequest>>> result = bidder.makeHttpRequests(bidRequest);
+
+        // then
+        assertThat(result.getValue()).extracting(HttpRequest::getUri)
+                .containsExactly(String.format(URL_TEMPLATE,
+                        HttpUtil.encodeUrl("placement id"), HttpUtil.encodeUrl("account id")));
+        assertThat(result.getErrors()).isEmpty();
+    }
+
+    @Test
+    public void makeHttpRequestsShouldCreateSingleRequest() {
+        // given
+        final BidRequest bidRequest = givenBidRequest(
+                givenImp("accountId", "placementId"),
+                givenImp(identity()));
+
+        // when
+        final Result<List<HttpRequest<BidRequest>>> result = bidder.makeHttpRequests(bidRequest);
+
+        // then
+        assertThat(result.getValue()).hasSize(1)
+                .extracting(HttpRequest::getPayload)
+                .flatExtracting(BidRequest::getImp)
+                .hasSize(2);
+        assertThat(result.getErrors()).isEmpty();
+    }
+
+    @Test
+    public void makeBidsShouldReturnErrorIfResponseBodyCouldNotBeParsed() {
+        // given
+        final HttpCall<BidRequest> httpCall = givenHttpCall("Incorrect body", null);
+
+        // when
+        final Result<List<BidderBid>> result = bidder.makeBids(httpCall, null);
+
+        // then
+        assertThat(result.getValue()).isEmpty();
+        assertThat(result.getErrors()).containsExactly(BidderError.badServerResponse("Bad server response."));
+    }
+
+    @Test
+    public void makeBidsShouldReturnErrorIfBidResponseIsNull() {
+        // given
+        final HttpCall<BidRequest> httpCall = givenHttpCall("null", null);
+
+        // when
+        final Result<List<BidderBid>> result = bidder.makeBids(httpCall, null);
+
+        // then
+        assertThat(result.getValue()).isEmpty();
+        assertThat(result.getErrors()).containsExactly(BidderError.badServerResponse("Empty SeatBid array"));
+    }
+
+    @Test
+    public void makeBidsShouldReturnErrorIfBidResponseSeatBidIsNull() throws JsonProcessingException {
+        // given
+        final HttpCall<BidRequest> httpCall = givenHttpCall(givenBidResponse(identity()), null);
+
+        // when
+        final Result<List<BidderBid>> result = bidder.makeBids(httpCall, null);
+
+        // then
+        assertThat(result.getValue()).isEmpty();
+        assertThat(result.getErrors()).containsExactly(BidderError.badServerResponse("Empty SeatBid array"));
+    }
+
+    @Test
+    public void makeBidsShouldReturnErrorIfBidResponseSeatBidIsEmpty() throws JsonProcessingException {
+        // given
+        final HttpCall<BidRequest> httpCall = givenHttpCall(givenBidResponse(response -> response
```

Comment:
Very minor. Please, place `givenBidResponse(response -> response` on the same line with `.seatbid(emptyList())), null`
It will be more clear where call to seatbid corresponds to. Also, please, check for similar occurences.